pkg/cmd/server/logs.go

```diff
@@ -33,22 +41,88 @@ var logsCmd = &cobra.Command{
 			query = "follow=true"
 		}
 
-		ws, res, err := apiclient.GetWebsocketConn("/log/server", &activeProfile, &query)
-		if err != nil {
-			log.Fatal(apiclient.HandleErrorResponse(res, err))
-		}
+		switch {
+		case fileFlag:
+			readServerLogFile()
+
+		default:
+
+			ws, res, err := apiclient.GetWebsocketConn("/log/server", &activeProfile, &query)
 
-		for {
-			_, msg, err := ws.ReadMessage()
 			if err != nil {
+				log.Error(apiclient.HandleErrorResponse(res, err))
+
+				if activeProfile.Id != "default" {
+					return
+				}
+
+				readLogsFile := true
+				form := huh.NewForm(
+					huh.NewGroup(
+						huh.NewConfirm().Description("An error occurred while connecting to the server. Would you like to read from local log file instead?").
+							Value(&readLogsFile),
+					),
+				).WithTheme(views.GetCustomTheme())
+				formErr := form.Run()
+				if formErr != nil {
+					log.Error(formErr)
```

Comment:
If there is a form error, you should log a fatal error that will exit the process.